cmd/gendocs/gendocs.go

```diff
@@ -25,19 +25,27 @@ func init() {
 
 // define things which go into the frontmatter
 type frontmatter struct {
-	Date        string
-	Title       string
-	Description string
-	Slug        string
-	URL         string
-	Source      string
+	Date              string
+	Title             string
+	Description       string
+	Slug              string
+	URL               string
+	Source            string
+	VersionIntroduced string
+	Status            string
 }
 
 var frontmatterTemplate = template.Must(template.New("frontmatter").Parse(`---
 title: "{{ .Title }}"
 description: "{{ .Description }}"
 slug: {{ .Slug }}
 url: {{ .URL }}
+{{- with .VersionIntroduced }}
```

Comment:
I was thinking you could pass the Annotations map in here directly and just iterate it

Something like

```
{{ range $key, $value := . Annotations }}
{{ $key }}: {{  $value }}
{{ end }}
```

Reply:
Ah yes, that is much more flexible. Pushed a change to this strategy.